src/components/modals/item-modal/PlaceBidModal.tsx

```diff
@@ -1,17 +1,23 @@
 import { Item } from '@types';
 import Button from '@ui/button';
 import Modal from 'react-bootstrap/Modal';
+import ClipLoader from 'react-spinners/ClipLoader';
 import useMetamask from '../../../features/auth/hooks/useMetamask';
 import { buyItem } from '../../../features/marketplace/store/marketplace.actions';
 import useAppDispatch from '../../../store/hooks/useAppDispatch';
+import useAppSelector from '../../../store/hooks/useAppSelector';
+import { selectNftState } from '../../../features/leda-nft/store/leda-nft.slice';
 
 type Props = {
   show: boolean;
   handleModal: () => void;
   item?: Item;
 };
 
+const Spinner = () => <ClipLoader className="spinner" color="#fff" size={18} />;
```

Comment:
Does the spinner color white look good in both light and dark modes?

Reply:
Hmm I think so